### PR TITLE
photon: added images for dev branch

### PR DESCRIPTION
**Pull request by YustasSwamp**
for x86_64 and aarch64 architectures.

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 0d73f46..8744d22 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,2 +1,4 @@
 photon:1.0-20180816 @ amd64
+photon:dev-20180901 @ amd64
+photon:dev-20180904 @ arm64v8
 photon:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index f813083..bad9afc 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -2,4 +2,7 @@ photon:1.0
 photon:1.0-20180816
 photon:2.0
 photon:2.0-20180823
+photon:dev
+photon:dev-20180901
+photon:dev-20180904
 photon:latest
diff --git a/_bashbrew-shared-tags b/_bashbrew-shared-tags
index e69de29..71ca3b3 100644
--- a/_bashbrew-shared-tags
+++ b/_bashbrew-shared-tags
@@ -0,0 +1 @@
+photon:dev -- photon:dev-20180901, photon:dev-20180904
diff --git a/photon_latest/Dockerfile b/photon_dev-20180901/Dockerfile
similarity index 34%
copy from photon_latest/Dockerfile
copy to photon_dev-20180901/Dockerfile
index 94b6372..f2888d8 100644
--- a/photon_latest/Dockerfile
+++ b/photon_dev-20180901/Dockerfile
@@ -1,8 +1,8 @@
 FROM scratch
-ADD photon-rootfs-2.0-bae8823.tar.bz2 /
+ADD photon-rootfs-2.0-7ca95e0.tar.bz2 /
 
-LABEL name="Photon OS 2.0 Base Image" \
+LABEL name="Photon OS Dev Base Image" \
     vendor="VMware" \
-    build-date="20180823"
+    build-date="20180901"
 
 CMD ["/bin/bash"]
diff --git a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2 b/photon_dev-20180901/photon-rootfs-2.0-7ca95e0.tar.bz2
similarity index 60%
copy from photon_latest/photon-rootfs-2.0-bae8823.tar.bz2
copy to photon_dev-20180901/photon-rootfs-2.0-7ca95e0.tar.bz2
index f47042b..1105a4c 100644
Binary files a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2 and b/photon_dev-20180901/photon-rootfs-2.0-7ca95e0.tar.bz2 differ
diff --git a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t' b/photon_dev-20180901/photon-rootfs-2.0-7ca95e0.tar.bz2  'tar -t'
similarity index 99%
copy from photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t'
copy to photon_dev-20180901/photon-rootfs-2.0-7ca95e0.tar.bz2  'tar -t'
index ecd35a5..382ed24 100644
--- a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t'	
+++ b/photon_dev-20180901/photon-rootfs-2.0-7ca95e0.tar.bz2  'tar -t'	
@@ -163,12 +163,8 @@ etc/ssl/certs/5620c4aa.0
 etc/ssl/certs/5620c4aa.pem
 etc/ssl/certs/56657bde.0
 etc/ssl/certs/56657bde.pem
-etc/ssl/certs/57bbd831.0
-etc/ssl/certs/57bbd831.pem
 etc/ssl/certs/57bcb2da.0
 etc/ssl/certs/57bcb2da.pem
-etc/ssl/certs/592c0a9a.0
-etc/ssl/certs/592c0a9a.pem
 etc/ssl/certs/5a4d6896.0
 etc/ssl/certs/5a4d6896.pem
 etc/ssl/certs/5ad8a5d6.0
@@ -211,8 +207,6 @@ etc/ssl/certs/76faf6c0.0
 etc/ssl/certs/76faf6c0.pem
 etc/ssl/certs/7719f463.0
 etc/ssl/certs/7719f463.pem
-etc/ssl/certs/790a7190.0
-etc/ssl/certs/790a7190.pem
 etc/ssl/certs/7992b8bb.0
 etc/ssl/certs/7992b8bb.pem
 etc/ssl/certs/7d0b38bd.0
@@ -744,7 +738,6 @@ usr/lib/libgssapi_krb5.so.2.2
 usr/lib/libgssrpc.so
 usr/lib/libgssrpc.so.4
 usr/lib/libgssrpc.so.4.2
-usr/lib/libhawkey.so.2
 usr/lib/libhistory.so.7
 usr/lib/libhistory.so.7.0
 usr/lib/libk5crypto.so
diff --git a/photon_latest/Dockerfile b/photon_dev-20180904/Dockerfile
similarity index 33%
copy from photon_latest/Dockerfile
copy to photon_dev-20180904/Dockerfile
index 94b6372..9977cc8 100644
--- a/photon_latest/Dockerfile
+++ b/photon_dev-20180904/Dockerfile
@@ -1,8 +1,8 @@
 FROM scratch
-ADD photon-rootfs-2.0-bae8823.tar.bz2 /
+ADD photon-rootfs-2.0-da347eda.tar.bz2 /
 
-LABEL name="Photon OS 2.0 Base Image" \
+LABEL name="Photon OS Dev Base Image" \
     vendor="VMware" \
-    build-date="20180823"
+    build-date="20180904"
 
 CMD ["/bin/bash"]
diff --git a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2 b/photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2
similarity index 53%
copy from photon_latest/photon-rootfs-2.0-bae8823.tar.bz2
copy to photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2
index f47042b..d36032a 100644
Binary files a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2 and b/photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2 differ
diff --git a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t' b/photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2  'tar -t'
similarity index 99%
copy from photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t'
copy to photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2  'tar -t'
index ecd35a5..de5bef5 100644
--- a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t'	
+++ b/photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2  'tar -t'	
@@ -163,12 +163,8 @@ etc/ssl/certs/5620c4aa.0
 etc/ssl/certs/5620c4aa.pem
 etc/ssl/certs/56657bde.0
 etc/ssl/certs/56657bde.pem
-etc/ssl/certs/57bbd831.0
-etc/ssl/certs/57bbd831.pem
 etc/ssl/certs/57bcb2da.0
 etc/ssl/certs/57bcb2da.pem
-etc/ssl/certs/592c0a9a.0
-etc/ssl/certs/592c0a9a.pem
 etc/ssl/certs/5a4d6896.0
 etc/ssl/certs/5a4d6896.pem
 etc/ssl/certs/5ad8a5d6.0
@@ -211,8 +207,6 @@ etc/ssl/certs/76faf6c0.0
 etc/ssl/certs/76faf6c0.pem
 etc/ssl/certs/7719f463.0
 etc/ssl/certs/7719f463.pem
-etc/ssl/certs/790a7190.0
-etc/ssl/certs/790a7190.pem
 etc/ssl/certs/7992b8bb.0
 etc/ssl/certs/7992b8bb.pem
 etc/ssl/certs/7d0b38bd.0
@@ -696,7 +690,7 @@ usr/lib/krb5/plugins/preauth/test.so
 usr/lib/krb5/plugins/tls/
 usr/lib/krb5/plugins/tls/k5tls.so
 usr/lib/ld-2.26.so
-usr/lib/ld-linux-x86-64.so.2
+usr/lib/ld-linux-aarch64.so.1
 usr/lib/libanl-2.26.so
 usr/lib/libanl.so
 usr/lib/libanl.so.1
@@ -744,7 +738,6 @@ usr/lib/libgssapi_krb5.so.2.2
 usr/lib/libgssrpc.so
 usr/lib/libgssrpc.so.4
 usr/lib/libgssrpc.so.4.2
-usr/lib/libhawkey.so.2
 usr/lib/libhistory.so.7
 usr/lib/libhistory.so.7.0
 usr/lib/libk5crypto.so
@@ -778,9 +771,6 @@ usr/lib/libmenuw.so.6
 usr/lib/libmenuw.so.6.0
 usr/lib/libm.so
 usr/lib/libm.so.6
-usr/lib/libmvec-2.26.so
-usr/lib/libmvec.so
-usr/lib/libmvec.so.1
 usr/lib/libncurses.so.6
 usr/lib/libncursesw.so.6
 usr/lib/libncursesw.so.6.0
```

</details>

File: library/photon [hunk 1]
```diff
@@ -11,3 +14,14 @@ GitCommit: 3c1e91142077d224ce57f8890e3ae12f64a92981
 Tags: 1.0, 1.0-20180816
 GitFetch: refs/heads/1.0-20180816
 GitCommit: fda590b062d1ac4d575c67fff4847b483f5097cd
+
+Tags: dev-20180901
+SharedTags: dev
+GitFetch: refs/heads/x86_64/dev-20180901
+GitCommit: 9dda8da4767f493b80ac75a5f52d783c49f27718
+
+Tags: dev-20180904
```
Comment (tianon) [inline library/photon]: This seems really strange (and is likely to be really confusing for users); is there any particular reason one of these can't be regenerated so they've got matching datestamps?

What this is going to look like as-is is that the page at https://hub.docker.com/_/photon/ will list both tags under the list of supported tags, with no obvious indication that each supports an entirely separate architecture (so users on `amd64` are likely to try running `dev-20180904` and it'll just fail with a cryptic error message about unsupported platform).

**Comment by YustasSwamp** [comment]
We build and publish x86_64 and aarch64 updates independently. That is the reason for different timestamps: like dev-20180904 for aarch64 and dev-20180901 for x86_64, but common tag is dev, which will point to latest dev-* for every architecture.
In result, I would like to see https://hub.docker.com/r/amd64/photon/ page having:

    2.0, 2.0-20180823, latest (docker/Dockerfile)
    1.0, 1.0-20180816 (docker/Dockerfile)
    dev, dev-20180901 (docker/Dockerfile)

and https://hub.docker.com/r/arm64v8/photon/ page having only:
    dev, dev-20180904 (docker/Dockerfile)

For example, ubuntu pages for different architectures are not in sync:
https://hub.docker.com/r/amd64/ubuntu/
https://hub.docker.com/r/arm64v8/ubuntu/

Ready for you advise here.
Thanks!

**Comment by tianon** [comment]
Ubuntu is a temporary out-of-sync issue -- what you'll see instead will be this: (which hopefully makes it much more obvious why I don't think this is a great solution for your users)

On https://hub.docker.com/_/photon/:

---

# Supported tags and respective `Dockerfile` links

## Simple Tags

-	[`2.0`, `2.0-20180823`, `latest` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/3c1e91142077d224ce57f8890e3ae12f64a92981/docker/Dockerfile)
-	[`1.0`, `1.0-20180816` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/fda590b062d1ac4d575c67fff4847b483f5097cd/docker/Dockerfile)
-	[`dev-20180901` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/9dda8da4767f493b80ac75a5f52d783c49f27718/docker/Dockerfile)
-	[`dev-20180904` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/0ab3eef33494bb18fe39f88985691d9e40447c51/docker/Dockerfile)

## Shared Tags

-	`dev`:
	-	[`dev-20180901` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/9dda8da4767f493b80ac75a5f52d783c49f27718/docker/Dockerfile)
	-	[`dev-20180904` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/0ab3eef33494bb18fe39f88985691d9e40447c51/docker/Dockerfile)

...

---

On https://hub.docker.com/r/amd64/photon/:

---

# Supported tags and respective `Dockerfile` links

## Simple Tags

-	[`2.0`, `2.0-20180823`, `latest` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/3c1e91142077d224ce57f8890e3ae12f64a92981/docker/Dockerfile)
-	[`1.0`, `1.0-20180816` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/fda590b062d1ac4d575c67fff4847b483f5097cd/docker/Dockerfile)
-	[`dev-20180901` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/9dda8da4767f493b80ac75a5f52d783c49f27718/docker/Dockerfile)

## Shared Tags

-	`dev`:
	-	[`dev-20180901` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/9dda8da4767f493b80ac75a5f52d783c49f27718/docker/Dockerfile)

...

---

On https://hub.docker.com/r/arm64v8/photon/:

---

# Supported tags and respective `Dockerfile` links

## Simple Tags

-	[`dev-20180904` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/0ab3eef33494bb18fe39f88985691d9e40447c51/docker/Dockerfile)

## Shared Tags

-	`dev`:
	-	[`dev-20180904` (*docker/Dockerfile*)](https://github.com/vmware/photon-docker-image/blob/0ab3eef33494bb18fe39f88985691d9e40447c51/docker/Dockerfile)

...

**Comment by YustasSwamp** [comment]
@tianon Thanks for clarification! Updated pull request.

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 0d73f46..da6520f 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,2 +1,4 @@
 photon:1.0-20180816 @ amd64
+photon:dev-20180904 @ amd64
+photon:dev-20180904 @ arm64v8
 photon:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index f813083..03ca6f8 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -2,4 +2,6 @@ photon:1.0
 photon:1.0-20180816
 photon:2.0
 photon:2.0-20180823
+photon:dev
+photon:dev-20180904
 photon:latest
diff --git a/photon_latest/Dockerfile b/photon_dev-20180904/Dockerfile
similarity index 34%
copy from photon_latest/Dockerfile
copy to photon_dev-20180904/Dockerfile
index 94b6372..f2888d8 100644
--- a/photon_latest/Dockerfile
+++ b/photon_dev-20180904/Dockerfile
@@ -1,8 +1,8 @@
 FROM scratch
-ADD photon-rootfs-2.0-bae8823.tar.bz2 /
+ADD photon-rootfs-2.0-7ca95e0.tar.bz2 /
 
-LABEL name="Photon OS 2.0 Base Image" \
+LABEL name="Photon OS Dev Base Image" \
     vendor="VMware" \
-    build-date="20180823"
+    build-date="20180901"
 
 CMD ["/bin/bash"]
diff --git a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2 b/photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2
similarity index 60%
copy from photon_latest/photon-rootfs-2.0-bae8823.tar.bz2
copy to photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2
index f47042b..1105a4c 100644
Binary files a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2 and b/photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2 differ
diff --git a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t' b/photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2  'tar -t'
similarity index 99%
copy from photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t'
copy to photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2  'tar -t'
index ecd35a5..382ed24 100644
--- a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t'	
+++ b/photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2  'tar -t'	
@@ -163,12 +163,8 @@ etc/ssl/certs/5620c4aa.0
 etc/ssl/certs/5620c4aa.pem
 etc/ssl/certs/56657bde.0
 etc/ssl/certs/56657bde.pem
-etc/ssl/certs/57bbd831.0
-etc/ssl/certs/57bbd831.pem
 etc/ssl/certs/57bcb2da.0
 etc/ssl/certs/57bcb2da.pem
-etc/ssl/certs/592c0a9a.0
-etc/ssl/certs/592c0a9a.pem
 etc/ssl/certs/5a4d6896.0
 etc/ssl/certs/5a4d6896.pem
 etc/ssl/certs/5ad8a5d6.0
@@ -211,8 +207,6 @@ etc/ssl/certs/76faf6c0.0
 etc/ssl/certs/76faf6c0.pem
 etc/ssl/certs/7719f463.0
 etc/ssl/certs/7719f463.pem
-etc/ssl/certs/790a7190.0
-etc/ssl/certs/790a7190.pem
 etc/ssl/certs/7992b8bb.0
 etc/ssl/certs/7992b8bb.pem
 etc/ssl/certs/7d0b38bd.0
@@ -744,7 +738,6 @@ usr/lib/libgssapi_krb5.so.2.2
 usr/lib/libgssrpc.so
 usr/lib/libgssrpc.so.4
 usr/lib/libgssrpc.so.4.2
-usr/lib/libhawkey.so.2
 usr/lib/libhistory.so.7
 usr/lib/libhistory.so.7.0
 usr/lib/libk5crypto.so
````

</details>

<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 0d73f46..da6520f 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,2 +1,4 @@
 photon:1.0-20180816 @ amd64
+photon:dev-20180904 @ amd64
+photon:dev-20180904 @ arm64v8
 photon:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index f813083..03ca6f8 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -2,4 +2,6 @@ photon:1.0
 photon:1.0-20180816
 photon:2.0
 photon:2.0-20180823
+photon:dev
+photon:dev-20180904
 photon:latest
diff --git a/photon_latest/Dockerfile b/photon_dev-20180904/Dockerfile
similarity index 33%
copy from photon_latest/Dockerfile
copy to photon_dev-20180904/Dockerfile
index 94b6372..9977cc8 100644
--- a/photon_latest/Dockerfile
+++ b/photon_dev-20180904/Dockerfile
@@ -1,8 +1,8 @@
 FROM scratch
-ADD photon-rootfs-2.0-bae8823.tar.bz2 /
+ADD photon-rootfs-2.0-da347eda.tar.bz2 /
 
-LABEL name="Photon OS 2.0 Base Image" \
+LABEL name="Photon OS Dev Base Image" \
     vendor="VMware" \
-    build-date="20180823"
+    build-date="20180904"
 
 CMD ["/bin/bash"]
diff --git a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2 b/photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2
similarity index 53%
copy from photon_latest/photon-rootfs-2.0-bae8823.tar.bz2
copy to photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2
index f47042b..d36032a 100644
Binary files a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2 and b/photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2 differ
diff --git a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t' b/photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2  'tar -t'
similarity index 99%
copy from photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t'
copy to photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2  'tar -t'
index ecd35a5..de5bef5 100644
--- a/photon_latest/photon-rootfs-2.0-bae8823.tar.bz2  'tar -t'	
+++ b/photon_dev-20180904/photon-rootfs-2.0-da347eda.tar.bz2  'tar -t'	
@@ -163,12 +163,8 @@ etc/ssl/certs/5620c4aa.0
 etc/ssl/certs/5620c4aa.pem
 etc/ssl/certs/56657bde.0
 etc/ssl/certs/56657bde.pem
-etc/ssl/certs/57bbd831.0
-etc/ssl/certs/57bbd831.pem
 etc/ssl/certs/57bcb2da.0
 etc/ssl/certs/57bcb2da.pem
-etc/ssl/certs/592c0a9a.0
-etc/ssl/certs/592c0a9a.pem
 etc/ssl/certs/5a4d6896.0
 etc/ssl/certs/5a4d6896.pem
 etc/ssl/certs/5ad8a5d6.0
@@ -211,8 +207,6 @@ etc/ssl/certs/76faf6c0.0
 etc/ssl/certs/76faf6c0.pem
 etc/ssl/certs/7719f463.0
 etc/ssl/certs/7719f463.pem
-etc/ssl/certs/790a7190.0
-etc/ssl/certs/790a7190.pem
 etc/ssl/certs/7992b8bb.0
 etc/ssl/certs/7992b8bb.pem
 etc/ssl/certs/7d0b38bd.0
@@ -696,7 +690,7 @@ usr/lib/krb5/plugins/preauth/test.so
 usr/lib/krb5/plugins/tls/
 usr/lib/krb5/plugins/tls/k5tls.so
 usr/lib/ld-2.26.so
-usr/lib/ld-linux-x86-64.so.2
+usr/lib/ld-linux-aarch64.so.1
 usr/lib/libanl-2.26.so
 usr/lib/libanl.so
 usr/lib/libanl.so.1
@@ -744,7 +738,6 @@ usr/lib/libgssapi_krb5.so.2.2
 usr/lib/libgssrpc.so
 usr/lib/libgssrpc.so.4
 usr/lib/libgssrpc.so.4.2
-usr/lib/libhawkey.so.2
 usr/lib/libhistory.so.7
 usr/lib/libhistory.so.7.0
 usr/lib/libk5crypto.so
@@ -778,9 +771,6 @@ usr/lib/libmenuw.so.6
 usr/lib/libmenuw.so.6.0
 usr/lib/libm.so
 usr/lib/libm.so.6
-usr/lib/libmvec-2.26.so
-usr/lib/libmvec.so
-usr/lib/libmvec.so.1
 usr/lib/libncurses.so.6
 usr/lib/libncursesw.so.6
 usr/lib/libncursesw.so.6.0
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #4807; c4f0b4a0; `arm64v8` (`photon`):

```console

$ bashbrew build photon:2.0
skipping "photon:2.0" (due to architecture "arm64v8"; only "amd64" supported)

$ test/run.sh photon:2.0
testing photon:2.0
	image does not exist!


$ bashbrew build photon:1.0
skipping "photon:1.0" (due to architecture "arm64v8"; only "amd64" supported)

$ test/run.sh photon:1.0
testing photon:1.0
	image does not exist!


$ bashbrew build photon:dev
Building bashbrew/cache:d6776e69a5621bc19ec07a50afc398bb40b504208442103a0a6f7c00fea200ca (photon:dev)
Tagging photon:dev
Tagging photon:dev-20180904

$ test/run.sh photon:dev
testing photon:dev
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed

```


The following images failed at least one test:

- `photon`:
  - `photon:2.0`
  - `photon:1.0`

Build test of #4807; c4f0b4a0; `amd64` (`photon`):

```console

$ bashbrew build photon:2.0
Using bashbrew/cache:3a82c7189d9ef02086efcd8f22938eba51397332bb508e6b39287cf4ec343a76 (photon:2.0)
Tagging photon:2.0
Tagging photon:2.0-20180823
Tagging photon:latest

$ test/run.sh photon:2.0
testing photon:2.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build photon:1.0
Using bashbrew/cache:8c9d0f167188f8637fddc4cb8a5cad2aaf832c2b2b4e78112d4954842820c5ba (photon:1.0)
Tagging photon:1.0
Tagging photon:1.0-20180816

$ test/run.sh photon:1.0
testing photon:1.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build photon:dev
Building bashbrew/cache:0a659f5db1daf4de76b3fc4f7a8dc8f5822f59f99138d8645303d5595d9134c5 (photon:dev)
Tagging photon:dev
Tagging photon:dev-20180904

$ test/run.sh photon:dev
testing photon:dev
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**